build/yamls/antrea-aks.yml

```diff
@@ -4124,6 +4127,11 @@ data:
     # 3. The Node IP
     #transportInterface:
 
+    # The names of the interfaces on Node which are connected to external devices to the antrea clusters.
```

Comment:
Just say: The names of the interfaces on Nodes that are used to forward multicast...

Reply:
update

Reply:
Reminder.

Reply:
updated